applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -103,19 +108,18 @@ const useStyles = makeStyles((theme) => ({
     "& .scrollbar": {
       paddingRight: 0,
       marginTop: theme.spacing(2),
-      [theme.breakpoints.up("sm")]: {
-        maxHeight: "calc(100vh - 19.5rem)",
-      },
-    },
-    [theme.breakpoints.up("sm")]: {
-      height: "calc(100vh)",
+      height: '100%',
     },
     "& .main-content": {
       padding: theme.spacing(3),
-      overflow: "auto",
-
-      [theme.breakpoints.up("sm")]: {
-        height: "calc(100%)",
+      height: '83vh',
```

Comment:
Still not ideal, not understanding why we are setting vertical height, I may be missing something

Reply:
I don't know how to fix the heights.